cosim/black-parrot-example/fpga/Makefile

```diff
@@ -1,10 +1,18 @@
 TOP  := $(shell git rev-parse --show-toplevel)
 CURR := $(shell pwd)
 
+BOARDNAME ?= ultra96v2
```

Comment:
I'd bring up the BOARDNAME to mk/Makefile.common so that all projects can inherit it and the PART

Reply:
There's a bunch of CFLAGS that are set per example and board in the example Makefile so maybe it's better to keep it there?